Tests/L0/Gradle/gradleGood.json

```diff
@@ -4,6 +4,21 @@
             "code": 0,
             "stdout": "Sample gradle output"
         },
+        "gradlew properties": {
+            "code": 0,
+            "stdout": "Something subprojects",
```

Comment:
it should be "subprojects []" if it's single module project. this mock response is imp to test the code